### PR TITLE
Charts

### PR DESCRIPTION
**Pull request by dusekpetr**

### CODE REVIEW

File: src/Repository/WorkRepository.php [hunk 1]
```diff
@@ -61,4 +61,22 @@ public function findClosedWorks($taskId) {
...
+            ->andWhere('w.startDate IS NULL')
+            ->getQuery()
+            ->getResult()
+            ;
+    }
+
+    public function findUserTimes($taskId) {
+        $sql = 'select u.mail, sum from (select user_id, SUM(strftime(\'%s\', w.end_date) - strftime(\'%s\', start_date)) sum from work w where w.task_id = :id group by user_id) join app_users u on (user_id=u.id)';
+        $params = array('id' => $taskId);
+
```
Comment (privrja) [inline src/Repository/WorkRepository.php:79]: ta mezera je zbytecna ne?

File: src/Repository/WorkRepository.php [hunk 2]
```diff
@@ -61,4 +61,22 @@ public function findClosedWorks($taskId) {
...
+        return $this->getEntityManager()->getConnection()->executeQuery($sql, $params)->fetchAll();
+    }
+}
```
Comment (privrja) [inline src/Repository/WorkRepository.php:82]: \n

File: templates/task/show.html.twig [hunk 3]
```diff
@@ -99,19 +99,26 @@
         </div>
     </div>
 
+
     <div class="row">
         <div class="col">
-            <canvas id="myChart"></canvas>
+            <canvas id="myChart" ></canvas>
```
Comment (privrja) [inline templates/task/show.html.twig:105]: Mozna by to chtelo lepsi nazev, pokud planujem mit vic tech grafu

**Review by privrja — APPROVED** [review]
Tak snad ok

File: templates/task/show.html.twig (hunk 3 above)
Comment (dusekpetr) [reply templates/task/show.html.twig:105]: nice to have 😄 2. iterace

**Pull request merged**